client/src/app.html

```diff
@@ -4,6 +4,10 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%sveltekit.assets%/favicon.png" />
     <meta name="viewport" content="width=device-width" />
+
+    <title>Cat Me if You Can</title>
+    <meta name="description" content="A social deduction game for 4-8 players with cats and puns." />
```

Comment:
This could potentially be replaced by the first paragraph in our rules modal since it's basically our game description

Reply:
implemented in c6ad548a0e6e80929e43d4895f29fa23d5cd4be2